_test/test_admin/test_upgrade_file_format.m

```diff
@@ -52,7 +52,7 @@ function test_upgrade_single_sqw_filebacked_noupgrade_range_warning(obj)
             w2  = sqw(targ_f{1});            
 
             [~,e] = lastwarn;
-            assertEqual(e,'HORACE:invalid_data_range');
+            assertEqual(e,'TESTS:my_warning');
```

Comment:
As this is going backwards in terms of proper format for errors, a comment saying why you have done this is needed

Reply:
Its not an error, its warning and we have not have other agreement on the warning format. 

I am slowely removing warnings about invalid data range when I am accessing invalid data range, if it is not crutial for the code operations, so the changes to test come when these additional warnings disappear